src/Chapter2.hs

```diff
@@ -620,7 +629,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: (Eq a, Num a) => [a] -> [a]
```

Comment:
Constraints here are redundant. Notice that you don't use any specific information about values of type `a`, so no need to require anything special here :slightly_smiling_face: 
```suggestion
takeEven :: [a] -> [a]
```